src/mgr/BaseMgrModule.cc

```diff
@@ -401,7 +401,7 @@ ceph_get_module_option(BaseMgrModule *self, PyObject *args)
   char *module = nullptr;
   char *key = nullptr;
   char *prefix = nullptr;
-  if (!PyArg_ParseTuple(args, "ssz:ceph_get_module_option", &module, &key,
+  if (!PyArg_ParseTuple(args, "ss|s:ceph_get_module_option", &module, &key,
```

Comment:
Changed back to ``s`` for the prefix parameter because it does not make sense to support ``None`` here if the parameter is optional now.